sample_structs_test.go

```diff
@@ -21,6 +21,7 @@ type SliceSample struct {
 
 type SliceStructSample struct {
 	Slice       []SliceStruct  `csv:"s,slice" csv[]:"2"`
+	Slice2      []SliceStruct  `csv:"sliceText"`
```

Comment:
Slice2 is just a slice of SliceStruct structs with only a `csv` tag, no `csv[]` expansion tag.